apps/web/playwright/new-booking-pages.e2e.ts

```diff
@@ -0,0 +1,134 @@
+import { expect } from "@playwright/test";
+
+import { test } from "./lib/fixtures";
+import {
+  bookFirstEvent,
+  bookOptinEvent,
+  bookTimeSlot,
+  selectFirstAvailableTimeSlotNextMonth,
+  selectSecondAvailableTimeSlotNextMonth,
+} from "./lib/testUtils";
+
+test.describe.configure({ mode: "parallel" });
+
+test.describe("free user", () => {
+  test.beforeEach(async ({ page, users, context }) => {
+    const free = await users.create();
+    context.addCookies([{ name: "new-booker-enabled", value: "true", url: "http://localhost:3000" }]);
```

Comment:
Test is the same test as the current booking page test, except that I need to enable the cookie. Apart from that all tests were green after adding the right test ids 🥳